RealtekAC97.txt

```diff
@@ -0,0 +1,18 @@
+[Section]
+Name = Realtek Audio Drivers for AC97 A4.06
+Version = A4.06 (5.10.0.6300)
+License = Propietary
+Description = Audio driver for Win2000/WinME/WinXP (WDM driver) for multiple AC97 cards like ALC850, ALC655 or ALC250. Suitable for multiple old workstation systems with VIA or C-media chipsets.
+Category = 13
+URLSite = https://www.realtek.com/
+URLDownload = https://akacc.net/win98/0001-WDM_A406.exe
+SHA1 = edca75940e8a30fb87c19278aaa36c3fbea02f00
+SizeBytes = 18734784
+Icon = realtek.ico
+
+[Section.amd64]
+URLDownload = https://akacc.net/win98/0001-WDM_A406.exe
```

Comment:
you do link the same exe for x86 and x64, which makes me think this is most likely an x86 binary. Sure, it might contain both versions of the driver, but if the exe runs only on x86 due to missing WOW64, then wouldn't it be wise to strip the amd64-block here entirely? Same question for RealtekHD.txt

Reply:
WOW64 is something that will be in the next... # days, but the first thing I need is to have the drivers. It could be .zip files, but .exe can be also extracted with 7zip, so it's ok.
This version contains both x86 and x64 drivers for Windows 2003 (32 and 64 bits). I know that they might not be the most functional ones... But they are interesting.

Reply:
stuff that you would just add to the [SECTION] but not in the [Section.amd64] wouldn't be downloadable at all in the ros x64 build? (then I would understand what you did)

Reply:
If I don't add the amd64, they won't show up. Yeah.